opentreemap/treemap/js/src/MapManager.js

```diff
@@ -150,25 +175,33 @@ function getBasemapLayers(config) {
             'Hybrid': makeBingLayer('AerialWithLabels')
         };
     } else if (config.instance.basemap.type === 'tms') {
-        layers = [L.tileLayer(config.instance.basemap.data, _ZOOM_OPTIONS)];
+        layers = [L.tileLayer(config.instance.basemap.data, MAX_ZOOM_OPTION)];
     } else {
-        return {'Streets': new L.Google('ROADMAP', _ZOOM_OPTIONS),
-                'Hybrid': new L.Google('HYBRID', _ZOOM_OPTIONS),
-                'Satellite': new L.Google('SATELLITE', _ZOOM_OPTIONS)};
+        return {'Streets': new L.Google('ROADMAP', MAX_ZOOM_OPTION),
+                'Hybrid': new L.Google('HYBRID', MAX_ZOOM_OPTION),
+                'Satellite': new L.Google('SATELLITE', MAX_ZOOM_OPTION)};
     }
     return layers;
 }
 
 function createPlotTileLayer(config) {
     var url = getPlotLayerURL(config, 'png'),
-        layer = L.tileLayer(url, _ZOOM_OPTIONS);
+        layer = L.tileLayer(url, MAX_ZOOM_OPTION);
+    makeLayerFilterable(layer, url, config);
+    return layer;
+}
+
+function createPolygonTileLayer(config) {
+    var url = getPlotLayerURL(config, 'png'),
```

Comment:
is this a typo? Do we want the plot url here, or the polygon url? This will be sent to `makeLayerFilterable`.